src/components/auth/enhanced-login-form.tsx

```
@@ -0,0 +1,274 @@
+
+      const result = await signIn(values.email, values.password)
+
+      if (result && result.error) {
+        setAuthError(getLocalizedErrorMessage(result.error))
+        return
+      }
+
+      // ログイン成功 - リダイレクト
+      router.push(redirectTo)
+    } catch (error) {
+      console.error('Login error:', error)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)